routes/deductions/deductions.controller.js

```diff
@@ -117,15 +120,52 @@ module.exports = function(app) {
     doRedirect,
   )
 
+  app.get('/trillium/longTermCare/ontario', renderWithData('deductions/trillium-longTermCare-ontario'))
+  app.post(
+    '/trillium/longTermCare/ontario',
+    checkSchema(trilliumlongTermCareOntarioSchema),
+    checkErrors('deductions/trillium-longTermCare-ontario'),
+    doYesNo('trilliumLongTermCareOntario', [
+      'trilliumLongTermCareTypeOntario',
+      'trilliumLongTermCareAmount',
+    ]),
+    doRedirect,
+  )
+
   app.get('/trillium/longTermCare/type', renderWithData('deductions/trillium-longTermCare-type'))
   app.post(
     '/trillium/longTermCare/type',
-    checkSchema(trilliumLongTermCareTypeSchema),
+    checkSchema(trilliumlongTermCareTypeSchema),
```

Comment:
Weird that this shows up as an addition/deletion because these lines are identical, right?

Reply:
The casing changed - there was some inconsistency with Long being spelled with a capital or lowercase. I made this one the same as the other schema.